examples/problems/1300_sequential_alignment.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<p>Add that we focus on the differences of policies in the AlignmentProblem. Please also include StarPolicy.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>